packages/matrix-client-server/src/account/3pid/3pid.test.ts

```diff
@@ -412,16 +707,17 @@ describe('Use configuration file', () => {
           ok: true,
           status: 200,
           // eslint-disable-next-line @typescript-eslint/promise-function-async
-          json: () =>
-            Promise.resolve({
+          json: () => {
+            return {
```

Comment:
I don't understand this change: the real API doesn't return a promise ?

Reply:
It does but in all mocks in identity service and in the mock right before this one there was no Promise.resolve for a similar API call, so I decided to remove it to match the rest of the code

Reply:
We need to reproduce exactly the express API everywhere in test, so if `req.json()` returns a promise, mock must do the same everywhere